README.md

```diff
@@ -402,6 +409,7 @@ docker run --rm --name builder \
     docker.io/unfoldedcircle/r2-pyinstaller:3.11.13  \
     bash -c \
       "python -m pip install -r requirements.txt && \
+      python -m pip install ./denonavrlib && \
```

Comment:
The GitHub action also specifies `PYTHONPATH=~/.local/lib/python\${PYTHON_VERSION}/site-packages:\$PYTHONPATH`.

The manual build works as documented here, but I didn't test the binary artifact yet.
If `PYTHONPATH` is not required it should be removed from the GitHub build. Otherwise it should also be in the manual build instruction.